src/test/config/test_config.c

```diff
@@ -0,0 +1,13 @@
+#include <unistd.h>
+#include <stdio.h>
+
+
+int main(int argc, char* argv[]) {
```

Comment:
It's not clear what the call to `gethostname` is testing. Assuming we're just trying to check that shadow parsed the config, I think we can just immediately exit. e.g. `{ return 0; }`